### PR TITLE
Issue #1, #2, #4, #5 fix

### PR DESCRIPTION
**Pull request by simantaturja**

### CODE REVIEW

File: src/main/java/com/example/crudrestapi/filters/JwtRequestFilter.java [hunk 1]
```diff
@@ -24,8 +30,33 @@
     @Autowired
     private LoginUserService loginUserService;
 
+    private String readInputStreamInStringFormat(InputStream stream, Charset charset) throws IOException {
```
Comment (JobaerChowdhury) [inline src/main/java/com/example/crudrestapi/filters/JwtRequestFilter.java:33]: I suggest we don't do it by ourselves, rather use some library function. For example apache commons IOUtils can convert this easily.

**Review by JobaerChowdhury — COMMENTED** [review]
Overall it looks way better than the previous version. 
Just found one issue and when that is handled I will approve this.

File: src/main/java/com/example/crudrestapi/filters/JwtRequestFilter.java (hunk 1 above)
Comment (simantaturja) [reply src/main/java/com/example/crudrestapi/filters/JwtRequestFilter.java:33]: sorry, that's a redundant code snippet. Didn't use it. Removing it

Comment (JobaerChowdhury) [reply src/main/java/com/example/crudrestapi/filters/JwtRequestFilter.java]: Got it. Looks better now.

**Review by JobaerChowdhury — APPROVED** [review]

**Pull request merged**